### PR TITLE
feat(core): Add metadata (customdata) to event log

### PR DESCRIPTION
**Pull request by flipswitchingmonkey**

### CODE REVIEW

File: packages/cli/src/InternalHooks.ts [hunk 1]
```diff
@@ -262,6 +264,9 @@ export class InternalHooks implements IInternalHooksClass {
 					isManual: executionMode === 'manual',
 					workflowId: workflowData?.id?.toString(),
 					workflowName: workflowData?.name,
+					metaData: executionMetadata?.map((metadata) => ({
```
Comment (krynble) [inline packages/cli/src/InternalHooks.ts:267]: This is gonna look like an array of key value pairs - is this what we expect?
We might want to use reduce instead if we want to have a single key-value object.

**Review by krynble — DISMISSED** [review]
LGTM - just left a question about the data format as it seems a bit odd to me

**Review by krynble — APPROVED** [review]
LGTM

**Comment by janober** [comment]
Got released with [`n8n@0.231.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.231.0)

**Pull request merged**